libclc/generic/include/spirv/spirv_builtins.h

```diff
@@ -12,6 +12,9 @@
 #ifndef CLC_SPIRV_BINDING
```

Comment:
Is this file auto-generated? Should we add a comment with instructions how to modify this file?

Reply:
yes it is auto-generated,

The PR for the generator is here https://github.com/intel/llvm/pull/1781 and it include a CMake rule to trigger the regeneration if needed. I'll add a warning header to it.

Reply:
I've made the update.

Reply:
@Naghasan, unfortunately, I don't see it.
Could you also take a look at CUDA LIT tests? They seem to hang in pre-merge checks.

Reply:
Sorry, I added it to the PR https://github.com/intel/llvm/pull/1781 as it adds the generator.